doc/man/salt.7

```diff
@@ -39706,7 +39706,10 @@ New in version 2018.3.0.
 .sp
 To automatically accept minions based on certain characteristics, e.g. the \fBuuid\fP
 you can specify certain grain values on the salt master. Minions with matching grains
-will have their keys automatically accepted.
+will have their keys automatically accepted. The grain can contain a ':' character in
```

Comment:
we should not be editing the `man` pages directly as these will be generated from the docs each release. This should generate from the doc/topics/tutorials/autoaccept_grains.rst file.

Reply:
Removal of changes done